Translators/WZDx/tim_generator.py

```diff
@@ -178,13 +178,21 @@ def vehicle_impact_supported(vehicle_impact):
 def get_first_road_name(feature):
     return feature["properties"]["core_details"]["road_names"][0]
 
+def get_start_date(feature):

```

Comment:
Why the change of start date?

Reply:
This is so the SDX doesn't drop TIMs that are being sent to it. Ryan let me know that with his updates if the start date and duration are both in the past the TIM is dropped - this prevents that from happening with the filtered list of active TIMs.

Reply:
I see, so we have an active workzone that's been active for a day and we set the duration time to be a half hour it would be rejected. This makes sense now, thanks!